src/main/java/ch/njol/skript/expressions/ExprOfflinePlayers.java

```diff
@@ -38,13 +38,9 @@
 import ch.njol.skript.lang.util.SimpleExpression;
 import ch.njol.util.Kleenean;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Offline players")
 @Description("All player that have ever joined the server.")
-@Examples({"loop all offline players:",
-		"	broadcast \"Player %loop-offlineplayer% has joined the server before!\""})
+@Examples({"send \"Size of all players who have joined the server: %size of all offline players%\""})
```

Comment:
Wrong example, this will not contain online players (as i remember from Skellett's code).

Reply:
Yes it will, offline players contains a list of every single player that has joined the server regardless if they're online.

Reply:
Then why it says **offline** players lol. It should return only offline players.